### PR TITLE
Change UI of the wiki results

### PR DESCRIPTION
**Pull request by asthas**

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -13,11 +13,20 @@ function App() {
...
-        type="search"
+        type='search'
         onInput={handleInput}
+        className='search-bar'
+        placeholder='Search wiki'
       />
-      {articles.map((article,i) => (
-        <li key={i}>{article}</li>
+      {articles.map((article) => (
+        Array.isArray(article)
```
Comment (shantanuraj) [inline src/App.js]: If you use the default value as an array in the `useFetch` callback you don't need this check for Array.isArray

File: src/App.js [hunk 2]
```diff
@@ -13,11 +13,20 @@ function App() {
...
+          ? article.map(item =>
+            item.length && item.includes('http')
```
Comment (shantanuraj) [inline src/App.js]: This check seems really obscure, can you add a comment about what's going on here?

Comment (asthas) [reply src/App.js]: This is for having only links in search results

File: src/App.js (hunk 1 above)
Comment (asthas) [reply src/App.js]: The first value of the response is the search term which is the string. SO i think this check is necessary

Comment (shantanuraj) [reply src/App.js]: No I meant if you did
```
  let articles = useFetch(
    `https://en.wikipedia.org/w/api.php?action=opensearch&origin=*&search=${searchTerm}`,
     []
  )
```

You don't need the Array.isArray check

File: src/App.js (hunk 2 above)
Comment (shantanuraj) [reply src/App.js]: Yeah a comment here to explain this would make it much better

**Pull request merged**